definitions/npm/react-native-thumbnail-video_v0.x.x/flow_v0.77.0-/test_react-native-thumbnail-video.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from "react";
+import { Thumbnail, type TYPES } from "react-native-thumbnail-video";
+import { getVideoId } from "react-native-thumbnail-video/src/helpers";
+
+var thumbnailVideo: React.ElementRef<typeof Thumbnail> = new Thumbnail();
```

Comment:
Would you mind wrapping test cases inside describe and it for better readability and maintainability? see https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope for details.